src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging/Strings.Designer.cs

```diff
@@ -1238,7 +1247,7 @@ internal class Strings {
                 return ResourceManager.GetString("RepositoryCountersignatureHasNoTimestamp", resourceCulture);
             }
         }
-
```

Comment:
Was this manually formatted?
Will adding a new string regenerate this without the spaces?

Reply:
I think the file was manually formatted previously.

The designer generated these changes.  (Compare with NuGet.Packaging/Strings.Designer.cs.)

If it's not a big deal, I'll keep these changes so future edits in the designer produce minimal diffs.

Reply:
> I'll keep these changes so future edits in the designer produce minimal diffs

Yep, 100%. 

We should keep whatever the designer generates.